File: app/src/main/java/com/xattacker/android/bingo/GradeRecorder.java
package com.xattacker.android.bingo;

public final class GradeRecorder
{
	private int _win, _lose;
	
	public GradeRecorder()
	{
		_win = _lose = 0;
	}
	
	public int getWinCount()
	{
		return _win;
	}
	
	public void addWinCount()
	{
		_win++;
	}
	
	public int getLostCount()
	{
		return _lose;
	}
	
	public void addLoseCount()
	{
		_lose++;
	}
}

File: app/src/main/java/com/xattacker/android/bingo/GradeRecorder.kt
package com.xattacker.android.bingo


interface GradeRecord
{
    val winCount: Int
    val loseCount: Int
}

class GradeRecorder : GradeRecord
{
    override var winCount: Int = 0
        private set

    override var loseCount: Int = 0
        private set

    fun addWin()
    {
        winCount++
    }

    fun addLose()
    {
        loseCount++
    }

    fun reset()
    {
        winCount = 0
        loseCount = 0
    }
}
